src/plugins/image_embeddable/public/image_editor/image_editor_flyout.tsx

```diff
@@ -0,0 +1,461 @@
+              }
+              fullWidth={true}
+              isInvalid={!!srcUrlError}
+              error={srcUrlError}
+            >
+              <EuiTextArea
+                data-test-subj={'imageEmbeddableEditorUrlInput'}
+                fullWidth
+                compressed={true}
+                placeholder={i18n.translate('imageEmbeddable.imageEditor.imageURLPlaceholderText', {
+                  defaultMessage: 'Example: https://elastic.co/my-image.png',
+                })}
+                value={srcUrl}
+                onChange={(e) => {
+                  const url = e.target.value;
+
+                  const { isValid, error } = props.validateUrl(url);
+                  if (!isValid) {
+                    setSrcUrlError(error!);
+                  } else {
+                    setSrcUrlError(null);
+                  }
+
+                  setSrcUrl(e.target.value);
+                }}
+              />
+            </EuiFormRow>
+          </>
+        )}
+        <EuiSpacer />
+        <EuiFormRow
+          label={
+            <FormattedMessage
+              id="imageEmbeddable.imageEditor.imageSizingLabel"
+              defaultMessage="Sizing"
+            />
+          }
+          helpText={
+            <FormattedMessage
+              id="imageEmbeddable.imageEditor.imageSizingHelpText"
+              defaultMessage="How the image should be resized relative to its container."
+            />
+          }
+          fullWidth
+        >
+          <EuiSelect
+            fullWidth
+            options={[
+              {
+                value: 'contain',
+                text: i18n.translate('imageEmbeddable.imageEditor.imageSizingContainOptionText', {
+                  defaultMessage: 'Fit maintaining aspect ratio',
+                }),
+              },
+              {
+                value: 'cover',
+                text: i18n.translate('imageEmbeddable.imageEditor.imageSizingCoverOptionText', {
+                  defaultMessage: 'Fill maintaining aspect ratio',
+                }),
+              },
+              {
+                value: 'fill',
+                text: i18n.translate('imageEmbeddable.imageEditor.imageSizingFillOptionText', {
+                  defaultMessage: 'Stretch to fill',
+                }),
+              },
+              {
+                value: 'none',
+                text: i18n.translate('imageEmbeddable.imageEditor.imageSizingNoneOptionText', {
+                  defaultMessage: "Don't resize",
+                }),
+              },
+            ]}
+            value={sizingObjectFit}
+            onChange={(e) =>
+              setSizingObjectFit(e.target.value as ImageConfig['sizing']['objectFit'])
+            }
+          />
+        </EuiFormRow>
+        <EuiSpacer />
+
+        <EuiFormRow
+          label={
+            <FormattedMessage
+              id="imageEmbeddable.imageEditor.imageBackgroundColorLabel"
+              defaultMessage="Background color"
```

Comment:
Could this be "Fill color"?

Reply:
I think `fill color` is not accurate, as we don't fill the image, but we fill the container behind the image. If image has transparent areas, then the container's fill color would be visible inside image's bounds. 

Maybe: "Container fill color" ?

Reply:
Will it be confusing to have both a "Fill mode" and a "Fill color"?  Maybe "Background color" is best.

I'm happy to review screenshots with the updated copy.